Research/ResearchUI/iOS/Views/RSDStepChoiceCell.swift

```diff
@@ -47,8 +47,8 @@ import UIKit
         }
     }
     
-    open private(set) var titleTextType: RSDDesignSystem.TextType = .body
-    open private(set) var detailTextType: RSDDesignSystem.TextType = .small
+    open private(set) var titleTextType: RSDDesignSystem.TextType = .small
```

Comment:
were the .small/.body text type defaults reversed/incorrect before? or did the meanings shift such that .small now means what .body used to and .bodyDetail means what .small used to?

Reply:
Nope, the rules changed. I was attempting with V1 to infer what they should be. Jordan and I spent the afternoon going through all the views in ResearchUI and updated them to match the fonts he'd been working thorough. 

The "body" maps to a larger text size now and the "small" is closer to the correct size. (both changed)

Unfortunately, we don't really have a way to version any of this  (*sigh*) since it is in the view and not in the font rules, but it's much more consistent.